x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_edit_extension.tsx

```diff
@@ -179,6 +183,10 @@ const WrappedPolicyDetailsForm = memo<{
               cardSize="m"
               customLink={policyTrustedAppsLink}
             />
+            <EuiSpacer size="s" />
+            {canSeeHostIsolationExeptions ? (
+              <FleetIntegrationHostIsolationExceptionsCard policyId={policyId} />
```

Comment:
As this component is checking the permissions inside I think you don't need this check here

Reply:
I thought about it and I'll go for your advice, remove the other permissions check.